client/state/analytics/middleware.js

```diff
@@ -15,12 +21,18 @@ const eventServices = {
 
 const pageViewServices = {
 	ga: ( { url, title } ) => analytics.ga.recordPageView( url, title ),
-	default: ( { url, title } ) => analytics.pageView.record( url, title )
+	'default': ( { url, title } ) => analytics.pageView.record( url, title ),
+};
+
+const loadTrackingTool = ( trackingTool, state ) => {
+	if ( trackingTool === 'Lucky Orange' && isTracking( state ) ) {
```

Comment:
The `isTracking` check needs to be negated -- we want to start tracking if we aren't yet, don't we 😉